config/CaloLayer2.json

```diff
@@ -1,8 +1,10 @@
 {
   "main_gdir": "DQMData/Run {0}/L1T/Run summary/L1TStage2CaloLayer2/",
   "hists": [{
+      "norm_type": "column",
       "path": "Central-Jets/*"
     }, {
+      "norm_type": "col",
```

Comment:
Should this be "column"?

Reply:
Ah, thought I changed this - so the pullvals.py plugin will accept "col" and "column" to select the norm_type.